clients/nasaClient.ts

```
@@ -0,0 +1,19 @@
+export default class nasaClient {
+
+    static getRoverList(): Promise<string | Rover[]> {
+        const apiUrl: string = `https://api.nasa.gov/mars-photos/api/v1/rovers?api_key=${apiKey}`
+        return axios.get(apiUrl)
+            .then((response: any): Rover[] => response.data.rovers)
+            .catch((error: any): string => {
+                console.error(error)
+                return `Error: ${error}`
+            })
+    }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)